projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-row.component.html

```diff
@@ -1,6 +1,6 @@
-<div (click)="toggle()" class="igx-grid__hierarchical-expander" [tabIndex]="tabindex" *ngIf="hasChildren" #expander>
-    <igx-icon *ngIf="!expanded" fontSet="material">expand_more</igx-icon>
-    <igx-icon *ngIf="expanded" fontSet="material">expand_less</igx-icon>
+<div (click)="toggle()" class="igx-grid__hierarchical-expander" [tabIndex]="tabindex" #expander>
```

Comment:
*ngIf="hasChildren"  is needed to show last child grids that have no child layout as flat (show no expander space). This should probably be restored.